phoenix-core/src/it/java/org/apache/phoenix/end2end/HashJoinIT.java

```diff
@@ -873,7 +873,7 @@ public void initTable() throws Exception {
                 "    SERVER AGGREGATE INTO DISTINCT ROWS BY [\"I.0:NAME\"]\n" +
                 "CLIENT MERGE SORT\n" +
                 "    PARALLEL LEFT-JOIN TABLE 0\n" +
-                "        CLIENT PARALLEL 1-WAY RANGE SCAN OVER " + MetaDataUtil.LOCAL_INDEX_TABLE_PREFIX + "" + JOIN_ITEM_TABLE_DISPLAY_NAME +" [-32768]\n" +
+                "        CLIENT PARALLEL 1-WAY RANGE SCAN OVER " +JOIN_ITEM_TABLE_DISPLAY_NAME +" [-32768]\n" +
```

Comment:
Yes it's using local indexes but it's confusing because we are having same table name. Only difference here is index id.  
What do you think of changing it to logical name in ExplainTable?
        buf.append("OVER " + tableRef.getTable().getPhysicalName().getString());